src/styles/_catalog.scss

```diff
@@ -1,15 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
   margin: 0 auto;
+  gap: 48px;
+
+  @media (min-width:487px) {
```

Comment:
Good job, the only thing I suggest is to replace `min-width:487px` to `max-width:487px`. Otherwise everything looks fine and clean =)